services/data_plane_manager/pom.xml

```diff
@@ -0,0 +1,192 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If services/data_plane_manager/extracted-media/media/image1.emf is a local file, remove the file.